ui/src/storage/local.ts

```diff
@@ -0,0 +1,23 @@
+import * as tanagra from "tanagra-api";
+import { registerStoragePlugin, StoragePlugin } from "./storage";
+
+const storageKey = "tanagra-userdata";
+
+@registerStoragePlugin()
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+class LocalStoragePlugin implements StoragePlugin {
+  async store(data: tanagra.UserData): Promise<void> {
+    localStorage.setItem(
```

Comment:
Possibly a future addition, but having a creation / lastModified timestamp on the individual resources in here might be quite useful, even if it isn't exposed to users directly.